server/src/test/java/org/eclipse/openvsx/search/DatabaseSearchServiceTest.java

```diff
@@ -33,9 +39,12 @@
 import org.springframework.data.util.Streamable;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
-import javax.persistence.EntityManager;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
 
 @ExtendWith(SpringExtension.class)
+@MockBean({
```

Comment:
You don't need `SimpleMeterRegistry` as `@MockBean`. The tests succeed without it.